pkgs/racket-doc/scribblings/reference/chaperones.scrbl

```diff
@@ -86,59 +86,19 @@ the same reason that they should avoid @racket[impersonator?].}
 Indicates whether @racket[v1] can be considered equivalent modulo
 impersonators to @racket[v2].
 
+Any two values that are @racket[eq?] to one another are also @racket[impersonator-of?].
 For values that include no impersonators, @racket[v1] and @racket[v2] can
 be considered impersonators of each other if they are @racket[equal?].
 
-Otherwise, impersonators within @racket[v2] must be intact within
-@racket[v1]:
+Otherwise, if @racket[v1] impersonates @racket[v1*] then @racket[(impersonator-of? v1 v2)]
+is @racket[#t] if and only if @racket[(impersonator-of? v1* v2)] is @racket[#t].
```

Comment:
Ok, more comments over there. Looking again, I see that part of the problem with the old documentation is that it's trying to centralize the definition of "impersonates" in `impersonator-of?`, but it works better to distribute that definition over things that construct impersonators. The main missing bit is the handling of compound values that contain impersonators, plus some details for immutable hash tables.